systems/data-plan/api-gateway/pkg/client/dataPlan.go

```diff
@@ -5,23 +5,20 @@ import (
 	"time"
```

Comment:
I think we can rename this file to `package.go`

Reply:
yeah I think I missed it somehow.